services/console-proxy/server/src/com/cloud/consoleproxy/NoVncConsoleHandler.java

```diff
@@ -0,0 +1,89 @@
+package com.cloud.consoleproxy;
+
+import com.cloud.consoleproxy.util.Logger;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.OutputStream;
+/**
+ * Created by root on 9/6/17.
+ */
+public class NoVncConsoleHandler extends AbstractHandler {
+    private static final Logger s_logger = Logger.getLogger(NoVncConsoleHandler.class);
+
+    public NoVncConsoleHandler() {
+
+    }
+
+    private void doHandle(Request request, HttpServletResponse httpServletResponse) throws IOException {
+        String queries = request.getUri().getQuery();
+        if (s_logger.isTraceEnabled())
+            s_logger.trace("Handle WebSocket Console request " + queries);
+
+
+        /* no authentication done here
+        *  authentication for the request would be added on websocket connection
+        *  initialization
+        */
+        String[] content =
+                new String[]{"<html>",
+                        "<head>",
+                        "<script type=\"text/javascript\" language=\"javascript\" src=\"/resource/js/jquery.js\"></script>",
+                        "<script> \n" +
+                                "    $(function(){\n" +
+                                "      $(\"#vncpage\").load(\"/resource/noVNC/novnc.html\"); \n" +
+                                "    });\n" +
+                                "    </script>",
+                        " <body> \n" +
+                                "     <div id=\"vncpage\"></div>\n" +
+                                "  </body> ",
+                        "</head>",
+                        "</html>"};
+
+        StringBuffer sb = new StringBuffer();
```

Comment:
This is static string, why not read/load from a resource of simply consume from the string itself, then have a string array and then use a stringbuffer to concat it?